### ISSUE TITLE
Request feature for Accounting

### ISSUE THREAD

**Issue by alifpramanarasa** [uilibrary/Stocky-Issues-and-Feature-request#26]

**Comment by naime-hossain** [comment]
Please add a clear description with more details otherwise we won't able to add your request to our Roadmap. thank you

**Comment by Stridesdata** [comment]
Good day,
If I got alif right, It may be similar to what I was about to request.

Add Chart of accounts
Add double-entry journal postings feature
Add Balance sheet report
Add Standard Accounting format income statement report
Add report for transaction history and Ledger posting history.

Will really appreciate this,... And there will be no rival app

**Comment by naime-hossain** [comment]
> Good day,
> If I got alif right, It may be similar to what I was about to request.
> 
> Add Chart of accounts
> Add double-entry journal postings feature
> Add Balance sheet report
> Add Standard Accounting format income statement report
> Add report for transaction history and Ledger posting history.
> 
> Will really appreciate this,... And there will be no rival app

We will try, If you like our item you can rete it . If you don’t know how to provide ratings & review please visit this link: https://help.market.envato.com/hc/en-us/articles/203269490-Giving-Feedback-On-Items Thank you

**Comment by alifpramanarasa** [comment]
> Good day,
> If I got alif right, It may be similar to what I was about to request.
> 
> Add Chart of accounts
> Add double-entry journal postings feature
> Add Balance sheet report
> Add Standard Accounting format income statement report
> Add report for transaction history and Ledger posting history.
> 
> Will really appreciate this,... And there will be no rival app

Right, like balance sheet, and other accounting module. Your app will over power and no rival

**Comment by naime-hossain** [comment]
Thank you we will discuss these feature and will try to implement. If you like our item you can rete it. If you don’t know how to provide ratings & review please visit this link: https://help.market.envato.com/hc/en-us/articles/203269490-Giving-Feedback-On-Items Thank you
